test/test_transforms.py

```diff
@@ -1074,6 +1075,26 @@ def test_rotate(self):
 
         self.assertTrue(np.all(np.array(result_a) == np.array(result_b)))
 
+    def test_rotate_fill(self):
+        img = F.to_pil_image(np.ones((100, 100, 3), dtype=np.uint8) * 255, "RGB")
+
+        modes = ("L", "RGB")
```

Comment:
I only tested these, since they are the most common. If you want me to add more here, let me know. One caveat: in its current state this test is not gonna work with `mode in ("LA", "RGBA")`. Unfortunately, the alpha channel is not filled with the same value as the other channels.

Reply:
Those modes are enough, thanks!